### PR TITLE
fix(ui): fix wrong text color

### PR DESCRIPTION
**Pull request by wa0x6e**
### Summary

Closes: #268 

Fix wrong text color

### How to test

1. Go to http://localhost:8080/#/s:test.wa0x6e.eth/proposal/0x504dd8da2930579b0aeb88946396140a74ce59fb8ccad5902cf1bfd2d33f1e07
2. The rank label in the choice form should have the correct text color on light and dark theme

Screenshot 2024-04-13 at 23 59 50
Screenshot 2024-04-13 at 23 59 54

### CODE REVIEW

**Review by bonustrack — APPROVED** [review]
utACK

**Review by bonustrack — CHANGES_REQUESTED** [review]
Wait

File: apps/ui/src/components/ProposalVoteRankedChoice.vue [hunk 1]
```diff
@@ -31,7 +31,7 @@ const selectedChoices = ref<number[]>(props.proposal.choices.map((_, i) => i + 1
             {{ proposal.choices[element - 1] }}
           </div>
           <div
-            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-white bg-skin-border px-2 text-center inline-block"
+            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-skin-text bg-skin-border px-2 text-center inline-block"
```
Comment (bonustrack) [inline apps/ui/src/components/ProposalVoteRankedChoice.vue:34]: It should use text-skin-link similar as the vote count
```suggestion
            class="h-[18px] min-w-[18px] rounded-full leading-[18px] text-[13px] text-skin-link bg-skin-border px-2 text-center inline-block"
```

**Review by bonustrack — APPROVED** [review]
tACK

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**